README.md

```diff
@@ -78,6 +82,20 @@ found in the `src/` directory.
     - stylesheets/
 ```
 
+`require('uswds')` will load all of Draft U.S. Web Design Standard's JavaScript onto the page. The uswds module itself does not export anything.
+
+The main Sass (SCSS) source file is here: 
+
+```
+node_modules/uswds/src/stylesheets/all.scss
+```
+
+The non-minified CSS that’s been precompiled is here:
```

Comment:
Well for `JS` a `require('uswds');` is all they would need to use for `npm`. There is no ~~good or~~ standard way of doing this with `CSS` so we are leaving it up for the developer's interpretation.